systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -332,14 +332,14 @@ protected void deleteInstalledYamls(File root, String namespace) {
         }
     }
 
-    protected void checkAllImages(JsonObject images) {
+    protected void checkAllImages(Map<String, String> images) {
         if (images.isEmpty()) {
             fail("There are no expected images");
         }
-        String zkImage = images.getString("zookeeper");
-        String kafkaImage = images.getString("kafka");
-        String tOImage = images.getString("topicOperator");
-        String uOImage = images.getString("userOperator");
+        String zkImage = images.get("zookeeper");
+        String kafkaImage = images.get("kafka");
+        String tOImage = images.get("topicOperator");
+        String uOImage = images.get("userOperator");
```

Comment:
Can these be as part of the `VersionModificationData`?

Reply:
i can add geters for those images, i wouldn't add variable just getter